SPARQLWrapper/KeyCaseInsensitiveDict.py

```diff
@@ -2,47 +2,46 @@
 
 """
 A simple implementation of a key case-insensitive dictionary.
-
 ..
   Developers involved:
-
   * Ivan Herman <http://www.ivan-herman.net>
   * Sergio Fernández <http://www.wikier.org>
   * Carlos Tejo Alonso <http://www.dayures.net>
   * Alexey Zakhlestin <https://indeyets.ru/>
-
   Organizations involved:
-
   * `World Wide Web Consortium <http://www.w3.org>`_
   * `Foundation CTIC <http://www.fundacionctic.org/>`_
-
   :license: `W3C® Software notice and license <http://www.w3.org/Consortium/Legal/copyright-software>`_
 """
 
+from typing import Dict, TypeVar
+
+K = str
+V = TypeVar("V")
```

Comment:
To avoid people confusing this with part of the public API of this module [[ref](https://docs.python.org/3/tutorial/classes.html#private-variables)] it may be best to rather prefix this with an underscore.

```suggestion
_V = TypeVar("_V")
```

Reply:
More on this:

https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces

> Even with __all__ set appropriately, internal interfaces (packages, modules, classes, functions, attributes or other names) should still be prefixed with a single leading underscore.

Reply:
7dc98eb